js/app/moltresEngine.js

```diff
@@ -196,6 +205,16 @@ placerat varius non id dui.',
         return Eknc.SetObjectModel.new_from_props(data);
     },
 
+    _generate_audio_object: function (data) {
+        data.synopsis = this._SYNOPSIS;
+        data.content_type = 'audio/webm';
+        data.title = AUDIO_TITLE;
+        data.ekn_id = IMAGES_DIR + VIDEO_FILENAME + new Array(this._audio_count++).fill('/').join('');
```

Comment:
I don't understand this. Am I reading correctly that it generates `resource:///com/endlessm/knowledge/data/images/tools/give_that_man_a_knighthood.webm/////////////`?

In any case if that's intended, you can use `'/'.repeat(this._audio_count++)`.